src/Templates/Public/scripts/default.js

```diff
@@ -2795,6 +2795,9 @@ function updateTextbox(action) {
     }
 
     txt.val(action.Value);
+    if (Number.isInteger(action.Size)) {
+        txt[0].rows = action.Size;
+    }
```

Comment:
The Size parameter is an `[int]` so the Number check isn't needed. It was also be worth checking that the textbox is a textarea as well, something like:

```javascript
if (action.Multiline && action.Size > 0) {
    txt[0].rows = action.Size;
}
```

Reply:
Fixed